apps/sms/js/attachment_renderer.js

```diff
@@ -51,6 +51,7 @@
        */
       renderTo: function(data, attachmentContainer) {
         attachmentContainer.classList.add(data.cssClass);
+        attachmentContainer.dataset.attachmentType = data.type;
```

Comment:
I don't think it's necessary to add one more type here. We already set the type within template here: https://github.com/jmcanterafonseca/gaia/blob/icon_mms_vcard/apps/sms/js/attachment_renderer.js#L279 It's not a proper way to have different type setting